src/main/java/org/teamresistance/frc/Robot.java

```diff
@@ -113,6 +117,8 @@ public void teleopPeriodic() {
     double orientation = IO.gyro.getAngle();
     SmartDashboard.putNumber("Gyro Angle", orientation);
 
+    opFlow.update();
```

Comment:
Make sure to update this to `IO.myOpticalFlowName.update()` if you're gonna be referencing OFS from `IO`.